File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {

        int n = nums.size();
        long sum = (n * (n+1))/2;
        int vector_sum = 0;
        for(int ele : nums){
            vector_sum += ele;
        }
        return sum - vector_sum;
    }
};

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int sum = n*(n+1)/2;
        int array_sum = 0;
        for(int ele : nums){
            array_sum += ele;
        }
        return sum - array_sum;
    }
}